dashboard/pages/screening/stages/s2_outliers_purging.py

```diff
@@ -81,19 +81,24 @@
         html.Div(
             className="d-flex flex-column gap-3 h-100",
             children=[
-                html.Div(
-                    className="overflow-auto mx-2 border border-3 rounded shadow bg-body-tertiary bg-primary flex-grow-1",
+                dcc.Loading(
```

Comment:
I think the loader is not needed when changing pages of bmg heatmaps (I would show it only on entering the stage)
The data is already loaded so the user wouldn't wait for anything anyway

Reply:
I see, the issue with `dcc.Loading` is that one cannot really specify if they want to have it loading only initially - there are some workarounds tho (eg. [this](https://community.plotly.com/t/loading-states-and-loading-component/19650/21)), so will try to incorporate it 🫡